.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: git checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```

Comment:
We should use `v2` for every `actions/checkout`. Unfortunately we have to explicitly do a checkout before every step, so this is used across the file.